### DISCUSSION TITLE
RHEL to AlmaLinux migration

### DISCUSSION THREAD

**Discussion by mdraganovic** [AlmaLinux/almalinux-deploy#11] [Q&A] [answered]
CentOS already has an check mark (kudos to the team) and Oracle Linux is on the roadmap. Any thoughts about supporting migration from RHEL?

**Comment by ezamriy** [comment, marked as answer]
I think we should do it, added it to the Roadmap.

**Reply by mdraganovic** [reply]
Yeah, I believe it makes a valid case. Thank you.
